pepper-apis/src/main/java/org/broadinstitute/ddp/event/HousekeepingTaskReceiver.java

```diff
@@ -160,6 +166,34 @@ private void handleElasticExport(PubsubMessage message, AckReplyConsumer reply)
         }
     }
 
+    private void handleUserUpdateDoNotContact(PubsubMessage message, AckReplyConsumer reply) {
```

Comment:
@MocanaAtBroad :
This works for Housekeeping pub/sub 
Does this approach work for DSM ?